test/Misc/TestFSharpInterfaces/TestFSharpInterfaces.fsproj

```diff
@@ -4,6 +4,12 @@
     <RootNamespace>TestFSharpInterfaces</RootNamespace>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
We probably don't need this because there's already the change in Directory.Build.props.